packages/p2p-v2/src/components/Modals/AdErrorTooltipModal/AdErrorTooltipModal.scss

```diff
@@ -0,0 +1,30 @@
+.p2p-v2-ad-error-tooltip-modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    width: 44rem;
+    padding: 2.4rem;
+    border-radius: 8px;
+    background: #fff;
+    box-shadow: 0px 32px 64px 0px rgba(14, 14, 14, 0.14);
```

Comment:
is it possible to create a mixin for these styles (excluding the width and padding) cause most of the modals are using the same ones 😓